server/models/Brand.js

```diff
@@ -0,0 +1,8 @@
+const {Schema, model} = require("mongoose");
+
+const Brand = new Schema({
```

Comment:
We talked we don't have brand in this project.

Reply:
ok, i'll delete
just thought about brand as an option for scaling the app